### DISCUSSION TITLE
Can't find published npm package

### DISCUSSION THREAD

**Discussion by aliksend** [community/community#145933] [npm]
I just published the npm package to my organization's github registry. 
Assume the package is `@mycoolorganization/mypackagename`
`npm publish` says that package is successfully published and I can `npm install` it, but I can't find this package in UI: neither in `https://github.com/aliksend?tab=packages`, nor in `https://github.com/orgs/mycoolorganization/packages`, nor in `https://github.com/mycoolorganization/mypackagename/packages`. I can see another packages in my organization, but not the one I'm looking for.

**Comment by jcf-junior** [comment]
Sometimes packages can take some time to show up.

Try executing `npm show <package-name>` and you can see all the versions of the package, so you can be confident it exists.

**Reply by aliksend** [reply]
I confirm that `npm show @mycoolorganization/mypackagename` shows information about my package, as well as `published 36 minutes ago by aliksend`.

But I'm not satisfied with contents of this package and I want to unpublish it.

Unfortunately I can't do it using `npm unpublish --force` because it says
```
405 Method Not Allowed - DELETE https://npm.pkg.github.com/@ mycoolorganization%2f mypackagename/-rev/undefined - Package deletion from the command line is disabled in GitHub Package Registry. For further details please visit https://help.github.com/en/packages/publishing-and-managing-packages/deleting-a-package.
```

If you assume that I need to wait a bit more to see my package in UI, I will wait.

**Reply by aliksend** [reply]
Just published another version of my package and both versions appeared in `https://github.com/orgs/mycoolorganization/packages`.

Thanks!

**Reply by jcf-junior** [reply]
***Edit: The answer I give you below is assuming the package is public, if it's private you might need to change its visibility to public first.

I'm assuming you want to delete the old version. 
If so do this:

1. On GitHub, navigate to the main page of the repository.
2. To the right of the list of files, click Packages.
3. Click the name of the package that you want to delete.
4. On the right, use the Edit package drop-down and select "Manage versions".
5. To the right of the version you want to delete, click Delete.
6. To confirm deletion, type the package name and click I understand the consequences, delete this version.
